### DISCUSSION TITLE
My account

### DISCUSSION THREAD

**Discussion by ChungHa136** [community/community#75528] [New to GitHub] [answered]
My main account @hoangchungk53qx1 has been flagged, please help me open it, I have made a lot of contributions to the community, I don't understand why my main account is locked, please look into it. help me

**Comment by ChungHa136** [comment]
Please help me. Github very importance to me,
@Akash1134

**Comment by hoangtien2k3** [comment]
He has contributed a lot to the community, please help him unlock his flagged account, please

**Reply by hoangchungk53qx1** [reply]
Please help me

**Comment by hoangchungk53qx1** [comment]
Please help me

**Comment by hoangtien2k3dev** [comment]
I see his great contributions to the github community, please help him get his account back from being flagged. @community

**Comment by AidenJamesYt** [comment, marked as answer]
several things:

1. using alt accounts to try to convince the mods to unflag is not a good thing.
2. dont give out ticket numbers, as it probably could end up to some personal info getting leaked.
3. pinging staff doesnt help either, as a ticket is meant for this exact reason.

**Comment by Akash1134** [comment]
👋 Hello @ChungHa136 , thank you for checking in. This account and topic has been flagged as: >One person or legal entity may maintain no more than one free Account.

Our private support team is the only one able to help with support requests on account flags. You will need to follow up with them directly through your open ticket.
